app/controllers/passengers_controller.rb

```diff
@@ -1,27 +1,26 @@
 class PassengersController < ApplicationController
   before_action :find_passenger, only: %i[show edit update destroy]
-
+  before_action :access_control, only: %i[destroy]
   def new
     @passenger = Passenger.new
   end
 
+  # Will be refactored
```

Comment:
If you right `TODO: refactor`, then CC will automatically flag it as an issue and then you can open a GH issue from that.